kops/kops.go

```diff
@@ -248,18 +210,18 @@ func (k *KopsCmd) GetKubeConfig(cluster clusteroperatorv1alpha1.KopsConfig) (clu
 
 	config := clusteroperatorv1alpha1.KubeConfig{}
 
-	kopsCmd := k.path +
+	kopsCmdStr := k.path +
 		" export kubecfg" +
 		" --name=" + cluster.Name +
 		" --state=" + cluster.StateStore +
 		" --kubeconfig=tmp/config-" + cluster.Name
```

Comment:
make `" --kubeconfig=" + viper.GetString("tmp.dir") +
		"/config-" + cluster.Name` to match what we pass to the function below